quotemachine/index.html

```diff
@@ -9,14 +9,14 @@
     <link rel="stylesheet" href="style.css">
 </head>
 
-<body class="random-color">
+<body class="random-color" style="transition:background-color 2s;">
```

Comment:
Please remove inline styles everywhere, only use extrenal styles in CSS (and modify them if needed in JS)